### ISSUE TITLE
Integration with ng2-ace-editor npm package

### ISSUE THREAD

**Issue by yadurvt** [convergencelabs/ace-collab-ext#17]
i'm looking to integrate collaborative plugin (npm install --save-dev @convergencelabs/ace-collab-ext) with angular 6's ng2-ace-editor(version 3.9). Package successfully imported but ends with an error message when trying to call
         `const curMgr = new AceCollabExt.AceMultiCursorManager(this.editor._editor.session);
          curMgr.addCursor("uid1", "User 1", "orange", {row: 2, column: 2});`
ng2-ace-editor is using brace for ace editor configuration. Is there any version mismatch? How can i rectify this. Could you please help me where i have gone wrong.

ng2-ace-editor is using brace for ace setup. how can i update this

**Comment by mmacfadden** [comment]
Can you show me what the error is that you see?  D you know what version of brace is being used?

**Comment by yadurvt** [comment]
Error shows at markerLayer.elt. inside update function. Brace is using ace editor version 1.2

**Comment by iwannabebot** [comment]
Any updates on this?

**Comment by mmacfadden** [comment]
Apologies for the delay. Some how missed the updates to this. I don't think brace is supported.  We used to use brace, but then got several requests to use a more modern version of ACE.  I think we switched over to using ace-builds directly which uses a more recent version.  

https://github.com/convergencelabs/ace-collab-ext/issues/7

I believe there were some API changes between ACE 1.2 and 1.4 that required some code changes to this library when we updated.  I suspect that the issue is due to an older version of ACE not being compatible.
